py/server/tests/test_updateby.py

```diff
@@ -154,6 +153,14 @@ def setUpClass(cls) -> None:
             rolling_wavg_time(ts_col="Timestamp", wcol="b", cols=["rwavg_b = b", "rwavg_e = e"], rev_time="PT00:00:10"),
             rolling_wavg_time(ts_col="Timestamp", wcol="b", cols=["rwavg_b = b", "rwavg_e = e"], rev_time=10_000_000_000, fwd_time=-10_000_000_00),
             rolling_wavg_time(ts_col="Timestamp", wcol="b", cols=["rwavg_b = b", "rwavg_e = e"], rev_time="PT30S", fwd_time="-PT00:00:20"),
+            # rolling formula
+            rolling_formula_tick(formula="sum(x)", formula_param="x", cols=["formula_a = a", "formula_d = d"], rev_ticks=10),
+            rolling_formula_tick(formula="avg(x)", formula_param="x", cols=["formula_a = a", "formula_d = d"], rev_ticks=10, fwd_ticks=10),
+            rolling_formula_time(formula="sum(x)", formula_param="x", ts_col="Timestamp", cols=["formula_b = b", "formula_e = e"], rev_time="PT00:00:10"),
+            rolling_formula_time(formula="avg(x)", formula_param="x", ts_col="Timestamp", cols=["formula_b = b", "formula_e = e"], rev_time=10_000_000_000,
+                             fwd_time=-10_000_000_00),
+            rolling_formula_time(formula="sum(x)", formula_param="x", ts_col="Timestamp", cols=["formula_b = b", "formula_e = e"], rev_time="PT30S",
```

Comment:
The `formula_param` variable makes me wonder what is going on under the covers.  Might there be a regex to fool?  How would something like this behave?

```python
 rolling_formula_tick(formula="sum(u)", formula_param="u", cols=["formula_a = a", "formula_d = d"], rev_ticks=10),
```

Reply:
The replacing call is `io.deephaven.engine.table.impl.select.FormulaUtil#replaceFormulaTokens` which doesn't leverage regex but does work hard to avoid over-replacements.  I tested `formula="sum(x)", formula_param="x"` and it handles it correctly.

Reply:
Also, providing aliases for the input columns  (`formula_a = a`) is handled properly.  The output column (after the formula is applied) is properly called `formula_a`

Reply:
great.